src/components/GeoPointEditor/GeoPointEditor.react.js

```diff
@@ -83,6 +83,34 @@ export default class GeoPointEditor extends React.Component {
   render() {
     let onChange = (target, e) => {
       let value = e.target.value;
+
+      if (!validateNumeric(value)) {
+        var values = value.split(",");
+
+        if (values.length == 2) {
+          values = values.map(function(val) {
+            return val.trim();
+          });
+
+          if(values[0].length > 0 && validateNumeric(values[0])) {
+
+            if(values[1].length <= 0 || !validateNumeric(values[1])) {
+              this.setState({ ["latitude"]: values[0] });
```

Comment:
The `[` and `"` are unnecessary, you can just do `this.setState({ latitude: values[0] });`